src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const thumbs = document.getElementById('thumbs');
+const largeImg = document.getElementById('largeImg');
+
+thumbs.onclick = (image) => {
+  const thumbItem = image.target.closest('a');
+
+  if (!thumbItem) {
+    return;
+  }
+
+  largeImg.src = thumbItem.href;
+  largeImg.alt = thumbItem.title;
+  image.preventDefault();
```

Comment:
With this naming, it can confuse other developers. Please fix it
You can prevent the event, but not image